src/theme/font-awesome.config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Configuration file for font-awesome-webpack
+ *
+ * In order to keep the bundle size low in production,
+ * disable components you don't use.
```

Comment:
So has to let the other developers working on this project to know a little bit indept about the config that goes into the webpack bootstrap css loader